ext/normalize_iplist.c

```diff
@@ -138,6 +155,9 @@ static VALUE serialize(VALUE self __attribute__((unused)), VALUE in) {
         rb_str_concat(out, rb_ary_join(extras, rb_str_new(NULL, 0)));
     }
     qsort(RSTRING_PTR(rb_string_value(&out)), total, 5, compare_serialized);
+    char *outp = RSTRING_PTR(rb_string_value(&out));
```

Comment:
I think `rb_string_value` here and in the line above are unnecessary, but I need to verify that again.